src/lib/types.hpp

```diff
@@ -138,6 +138,9 @@ using AttributeVectorWidth = uint8_t;
 using PosList = pmr_vector<RowID>;
 using ColumnIDPair = std::pair<ColumnID, ColumnID>;
 
+using PluginHandle = void*;
+using PluginName = std::string;
```

Comment:
Personal preference: Wouldn't alias this. Every string variable is a "name" or a "text" and they do not all get their own type-alias

Reply:
I find it quite descriptive in the `unordered_map` that maps from plugin name to handle.